sdk_contrib/system_metrics/src/main/java/io/opentelemetry/sdk/contrib/metrics/system/SystemMetrics.java

```diff
@@ -0,0 +1,221 @@
+            new Callback<LongResult>() {
+              @Override
+              public void update(LongResult r) {
+                GlobalMemory mem = hal.getMemory();
+                r.observe(mem.getTotal(), TYPE_LABEL_KEY, "total");
+                r.observe(mem.getAvailable(), TYPE_LABEL_KEY, "available");
+              }
+            });
+
+    meter
+        .doubleValueObserverBuilder("system.cpu")
+        .setDescription("System CPU")
+        .setUnit("seconds")
+        .setConstantLabels(labels)
+        .build()
+        .setCallback(
+            new Callback<DoubleResult>() {
+              @Override
+              public void update(DoubleResult r) {
+                long[] cpuTicks = hal.getProcessor().getSystemCpuLoadTicks();
+                r.observe(cpuTicks[TickType.SYSTEM.getIndex()] * 1000, TYPE_LABEL_KEY, "system");
+                r.observe(cpuTicks[TickType.USER.getIndex()] * 1000, TYPE_LABEL_KEY, "user");
+                r.observe(cpuTicks[TickType.IDLE.getIndex()] * 1000, TYPE_LABEL_KEY, "idle");
+              }
+            });
+
+    meter
+        .longValueObserverBuilder("sys.net.bytes")
+        .setDescription("System network bytes")
+        .setUnit("bytes")
+        .setConstantLabels(labels)
+        .build()
+        .setCallback(
+            new Callback<LongResult>() {
+              @Override
+              public void update(LongResult r) {
+                long recv = 0;
+                long sent = 0;
+
+                for (NetworkIF networkIf : hal.getNetworkIFs()) {
+                  recv += networkIf.getBytesRecv();
+                  sent += networkIf.getBytesSent();
+                }
+
+                r.observe(recv, TYPE_LABEL_KEY, "bytes_recv");
+                r.observe(sent, TYPE_LABEL_KEY, "bytes_sent");
+              }
+            });
+
+    meter
+        .longValueObserverBuilder("runtime.java.mem")
+        .setDescription("Runtime memory")
+        .setUnit("bytes")
+        .setConstantLabels(labels)
+        .build()
+        .setCallback(
+            new Callback<LongResult>() {
+              @Override
+              public void update(LongResult r) {
+                processInfo.updateAttributes();
+                r.observe(processInfo.getResidentSetSize(), TYPE_LABEL_KEY, "rss");
+                r.observe(processInfo.getVirtualSize(), TYPE_LABEL_KEY, "vms");
+              }
+            });
+
+    meter
+        .doubleValueObserverBuilder("runtime.java.cpu")
+        .setDescription("Runtime CPU")
+        .setUnit("seconds")
+        .setConstantLabels(labels)
+        .build()
+        .setCallback(
+            new Callback<DoubleResult>() {
+              @Override
+              public void update(DoubleResult r) {
+                processInfo.updateAttributes();
+                r.observe(processInfo.getUserTime() * 1000, TYPE_LABEL_KEY, "user");
+                r.observe(processInfo.getKernelTime() * 1000, TYPE_LABEL_KEY, "system");
```

Comment:
JVM mbeans provide process cpu as a single number (i assume `system` + `user`?), no breakdown of `system`/`user` that i know of